packages/utils/README.md

```diff
@@ -0,0 +1,35 @@
+![Coverage](https://img.shields.io/codecov/c/github/openwallet-foundation-labs/sd-jwt-js)
```

Comment:
we should keep in mind that our coverage rate is calculated for the whole project, not for every library by itself.

I don't see the need to present it for each individual, just to mention it ;)

Reply:
Yeah, good point. Maybe we should put the [codecov link](https://app.codecov.io/gh/openwallet-foundation-labs/sd-jwt-js) of our project. To see the details
What do you think about it?

Reply:
I would remove anything from the packages readme that does not belong there, e.g. the code coverage of the project.

Reply:
Okay :) done in [900158b](https://github.com/openwallet-foundation-labs/sd-jwt-js/pull/102/commits/900158b533c41259609522aec7169063788f3f11)